src/Coordination/KeeperStateMachine.cpp

```diff
@@ -22,6 +22,7 @@ namespace
         ReadBufferFromNuraftBuffer buffer(data);
         KeeperStorage::RequestForSession request_for_session;
         readIntBinary(request_for_session.session_id, buffer);
+        readIntBinary(request_for_session.time, buffer);
```

Comment:
I think we can do it in a compatible way if we add time to the end of buffer after request body.

Reply:
@nicelulu what do you think? Otherwise old state machines will not be able to parse it.

Reply:
Yes, this is very ingenious.